views/blocks/sepa_mandate_modal.tpl

```diff
@@ -0,0 +1,51 @@
+[{*
+* Shop System Plugins:
+* - Terms of Use can be found under:
+* https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+* - License can be found under:
+* https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+*
+*}]
+
+<div id="mandateModal" class="modal">
+
+  [{assign var="shopAddress" value=$oViewConf->getShopAddress()}]
+
+  <div class="modal-content">
+    <span class="close-modal">&times;</span>
```

Comment:
Please check if it's necessary to add a `<div class="clearfix"></div>` after this element too.